### PR TITLE
fix to issue 84, modules work in interactive mode

### PR DESCRIPTION
**Pull request by Vaishaal**
Solves issue #84 

Relative paths were not handled properly when serializing modules.

### CODE REVIEW

File: tests/test_modules.py [hunk 1]
```diff
@@ -101,3 +102,15 @@ def func(x):
         for f in info['pkg_ver_list']:
             print(f[0])
 
+class InteractiveTest(unittest.TestCase):
+    def setUp(self):
```
Comment (ericmjonas) [inline tests/test_modules.py]: This wren executor isn't used, can we get rid of it?

File: tests/test_modules.py [hunk 2]
```diff
@@ -101,3 +102,15 @@ def func(x):
         for f in info['pkg_ver_list']:
             print(f[0])
 
+class InteractiveTest(unittest.TestCase):
```
Comment (ericmjonas) [inline tests/test_modules.py]: Can we add a little bit of documentation here on what we're trying to test, and explain the difference between interactive vs regular mode?

File: tests/test_modules.py [hunk 3]
```diff
@@ -101,3 +102,15 @@ def func(x):
...
+    def setUp(self):
+        self.wrenexec = pywren.default_executor()
+
+    def test_simple(self):
+        ret = os.system("python -c \"import pywren;import extmodule;pwex = pywren.default_executor();results = pwex.map(extmodule.foo_add, [0]);print(results[0].result())\"")
```
Comment (ericmjonas) [inline tests/test_modules.py]: According to the python docs: "The subprocess module provides more powerful facilities for spawning new processes and retrieving their results; using that module is preferable to using this function. See the Replacing Older Functions with the subprocess Module section in the subprocess documentation for some helpful recipes."

Can we try using `subprocess` instead?

File: tests/test_modules.py (hunk 2 above)
Comment (Vaishaal) [inline tests/test_modules.py]: fixed.

**Pull request merged**